src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/ApplicationInsightsServiceCollectionExtensions.cs

```diff
@@ -257,9 +271,9 @@ internal static LoggerFilterOptions CreateFilterOptions(LoggerFilterOptions regi
                     }
 
                     // QuickPulse can have a startup performance hit, so delay its initialization.
-                    delayer.ScheduleInitialization(() => module.Initialize(configuration), options.QuickPulseInitializationDelay);
+                    delayer.ScheduleInitialization(() => module.Initialize(configuration), options.LiveMetricsInitializationDelay);
                 }
-                else
+                else if (module != null)
```

Comment:
Can the module ever be null here? You can get the NullTelmetryModule.Instance, but that should just be a no-op.

Reply:
Yes, at least in our tests they can be null. One of the instance in `telemetryModules` was null. I think it's because the Host setup test explicitly removes an `ITelemetryModule`. I figured it's a harmless check to have.